MIAUI/ViewModels/LoginViewModel.cs

```diff
@@ -20,15 +23,16 @@ async void LogIn()
         {
             var result = await authService.LoginAsync(CancellationToken.None);
             authService.GetUserClaims(result, user);
-
             if (result is not null)
             {
+                App.UserRepo.AddNewPerson(user.Name, user.Email);
+                id = user.Id;
                 await App.Current.MainPage.DisplayAlert("Success", "Successfully logged in", "Ok");
-                await Shell.Current.GoToAsync($"{nameof(NewTaskPage)}",
-                new Dictionary<string, object>
-                {
-                    [nameof(User)] = user
-                });
+                await Shell.Current.GoToAsync($"{nameof(NewTaskPage)}?Id={id}",
+                    new Dictionary<string, object>
+                    {
+                        [nameof(User)] = user
```

Comment:
So are you passing in the id and user? (I didn't know until yesterday that you could pass non-strings as query parameters so that is cool)
But it doesn't look like the TaskViewModel is accepting the id you are passing in here. So maybe add that to the TaskViewModel or let just remind @gal42 and @LeandraHall to accept that